sklearn/impute.py

```diff
@@ -21,6 +22,7 @@
 from .utils.sparsefuncs import _get_median
 from .utils.validation import check_is_fitted
 from .utils.validation import FLOAT_DTYPES
+from .utils.fixes import custom_isnan
```

Comment:
I would rename it to `compat_isnan` or `safe_isnan`.

Reply:
Actually, I would even make this function explicitly private: `_compat_isnan` or `_safe_isnan`.